Emby.Server.Implementations/Services/ServiceExec.cs

```diff
@@ -1,3 +1,5 @@
+#pragma warning disable SA1649
```

Comment:
I think this should be fixed instead of disabled. If you think it's too big a change for this PR it can always go into a separate one later.

Reply:
It's more that we should probably just fix these later, when they are moved to the Jellyfin namespace. At least, that's what @Bond-009 told me.

Reply:
Right, if the intent is to fix these later then I think the #pragma should be removed so that the warning is still there. We should only suppress warnings for issues we do not intend to fix in the future

Reply:
See https://github.com/jellyfin/jellyfin/pull/2937/files/7afe23f366fbb0c62ad515577c2b7e5968d81b6e#r426119785